### PR TITLE
feat: results page added

### PR DESCRIPTION
**Pull request by xsaurabhsuthar**
Result Page
image

Text background color will be fixed when merged with dev.

### CODE REVIEW

**Review by shobit1337 — APPROVED** [review]
Except for colours, Everything else looks good.

File: css/result.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+.score__header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin: 1rem 0;
+}
+
+.question--container {
+  margin: 20px 0;
```
Comment (znjs) [inline css/result.css:12]: Try using relative units for responsiveness

**Review by znjs — APPROVED** [review]

**Comment by xsaurabhsuthar** [comment]
@shobit1337 are colors too bright ?Should I use light ones? or just the text background color?
image

File: css/result.css (hunk 1 above)
Comment (xsaurabhsuthar) [reply css/result.css:12]: Done.

**Pull request merged**